packages/cli/src/ui/App.tsx

```diff
@@ -444,6 +449,77 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
     [handleSlashCommand],
   );
 
+  const keystrokeHandlers = useMemo<KeystrokeHandler[]>(
+    () => [
+      {
+        input: 'o',
+        ctrl: true,
+        handler: () => setShowErrorDetails((prev) => !prev),
+      },
+      {
+        input: 't',
+        ctrl: true,
+        handler: () => {
+          const newValue = !showToolDescriptions;
+          setShowToolDescriptions(newValue);
+
+          const mcpServers = config.getMcpServers();
+          if (Object.keys(mcpServers || {}).length > 0) {
+            handleSlashCommand(newValue ? '/mcp desc' : '/mcp nodesc');
+          }
+        },
+      },
+      {
+        input: 'e',
+        ctrl: true,
+        condition: () => !!ideContext,
+        handler: () => setShowIDEContextDetail((prev) => !prev),
+      },
+      {
+        input: 'c',
+        ctrl: true,
+        shift: true,
```

Comment:
is shift: true intended? I thought this one should allow both with and without shift?